internal/testrunner/runners/system/runner.go

```diff
@@ -42,8 +42,8 @@ const (
 	testRunMaxID = 99999
 	testRunMinID = 10000
 
-	allFieldsBody = `{"fields": ["*"]}`
-	DevDeployDir  = "_dev/deploy"
+	checkFieldsBody = `{"fields": ["*"], "aggs": {"degraded": {"filter": {"exists": {"field": "_ignored"}}}}}`
```

Comment:
I think the returned error should at a minimum include the names of ignored fields. You could aggregate those field names. (It needs a runtime field to access the names as per https://github.com/elastic/elasticsearch/issues/59946#issuecomment-947796041).

```
POST logs-*/_search
{
  "size": 10,
  "runtime_mappings": {
    "my_ignored": {
      "type": "keyword",
      "script": {
        "source": "for (def v : params['_fields']._ignored.values) { emit(v); }"
      }
    }
  },
  "aggs": {
    "all_ignored": {
      "filter": {
        "exists": {
          "field": "_ignored"
        }
      },
      "aggs": {
        "ignore_fields": {
          "terms": {
            "size": 100,
            "field": "my_ignored"
          }
        }
      }
    }
  }
}
```

Reply:
As mentioned I wanted to wait with this until the change to make _ignored aggregatable is merged, but you are right that we don't need to block on it - I changed the code to use the runtime field workaround for now.